.github/workflows/lint_python.yml

```
@@ -0,0 +1,25 @@
+name: lint_python
+on: [pull_request, push]
```

We already have Python linting tools. Check logs of "Docs" builder: http://pullrequest.opencv.org/buildbot/builders/precommit_docs/builds/33024

Team doesn't have spare time to maintain duplicated functionality.

---

Could you please replace default trigger:

```
-on: [pull_request, push]
+on: workflow_dispatch
```

---

The only mention of `buildbot` in the code of this repo is related to Python 2.  Also, the repo has no [`.pre-commit-config.yaml `](https://pre-commit.com/) which is customary on projects that use pre-commit.  This would enable contributors to test locally as well as enable `buildbot` to run the exact same tests.
```suggestion
on: workflow_dispatch
```
The problem with making this change is that _contributors get no feedback_ on the success or failure of the tests on their submissions because [`workflow_dispatch` is a manual event](https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#manual-events) unlike `push` and `pull_request`.  Is there a way that you could connect the buildbot results to pull requests as numerous other tools do so that contributors get feedback on their submissions?